src/langsmith/user-management.mdx

```diff
@@ -347,28 +395,47 @@ Examples:
 * `Groups-Organization User:Engineering:Developers`
 * `Organization User:Marketing:Viewers`
 
-### Email verification
+### Mapping
 
-In cloud only, creating a new user with SCIM triggers an email to the user. They must verify their email address by clicking the link in this email. The link expires in 24 hours, and can be resent if needed by removing and re-adding the user via SCIM.
+While specific instructions depending on the identity provider may vary, these mappings show what is supported by the LangSmith SCIM integration:
 
-### Prerequisites
+#### User Attributes
 
-- Your organization must be on an Enterprise plan.
-- Your Identity Provider (IdP) must support SCIM 2.0.
-- Only [`Organization Admins`](/langsmith/observability-concepts#organization-roles) can configure SCIM.
-- For cloud customers: [SAML SSO](/langsmith/user-management) must be configured for your organization.
-- For self-hosted customers: [OAuth with Client Secret](/langsmith/self-host-sso#with-secret) authentication mode must be enabled.
+| **LangSmith App Attribute**    | **Identity Provider Attribute**                       | **Matching Precedence** |
+| ------------------------------ | ----------------------------------------------------- | ----------------------- |
+| `userName`<sup>1</sup>         | email address                                         |                         |
+| `active`                       | `!deactivated`                                        |                         |
+| `emails[type eq "work"].value` | email address<sup>2</sup>                             |                         |
+| `name.formatted`               | `displayName` OR `givenName + familyName`<sup>3</sup> |                         |
+| `givenName`                    | `givenName`                                           |                         |
+| `familyName`                   | `familyName`                                          |                         |
+| `externalId`                   | `sub`<sup>4</sup>                                     | 1                       |
 
-<Note>
-Note the following:
+1. `userName` is not required by LangSmith
+1. Email address is required
+1. Use the computed expression if your `displayName` does not match the format of `Firstname Lastname`
+1. To avoid inconsistency, this should match the SAML `NameID` assertion (in cloud) and the `sub` OAuth2.0 claim (in self-hosted).
```

Comment:
```suggestion
1. To avoid inconsistency, this should match the SAML `NameID` assertion for cloud customers, or the `sub` OAuth2.0 claim for self-hosted.
```